app/scripts/controllers/reviewRegistration.js

```diff
@@ -280,4 +329,117 @@ angular.module('confRegistrationWebApp')
     $scope.hasPendingCheckPayment = function(payments){
       return _.some(payments, { paymentType: 'CHECK', status: 'PENDING' });
     };
+
+    ////// START EVENT-433 //////
```

Comment:
@hlbraddock LOL Thanks Lee.  I added those comments to make it easier to see beginning and end.